vendor.conf

```diff
@@ -155,7 +155,7 @@ github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
 github.com/spf13/cobra v0.0.3
 github.com/spf13/pflag v1.0.1
 github.com/inconshreveable/mousetrap 76626ae9c91c4f2a10f34cad8ce83ea42c93bb75 # v1.0
-github.com/Nvveen/Gotty a8b993ba6abdb0e0c12b0125c603323a71c7790c https://github.com/ijc25/Gotty
+github.com/nvveen/gotty a8b993ba6abdb0e0c12b0125c603323a71c7790c https://github.com/ijc25/Gotty
```

Comment:
We can't change this; the repository itself is cased like this; see https://github.com/Nvveen/Gotty

There's a pull request being reviewed though that removes this dependency 


Also;  changing the name upstream would cause a lot of issues, eg see https://github.com/sirupsen/logrus/issues/451 and https://github.com/sirupsen/logrus/pull/384

Reply:
it still _is_ causing issues - and in new code:

I'm getting 

```
go: github.com/Sirupsen/logrus@v1.3.0: parsing go.mod: unexpected module path "github.com/sirupsen/logrus"
--
```

from trying to use `go mod` with docker/engine and docker/cli

Reply:
@SvenDowideit which repository are you fetching from? I think `go mod` (when pointing at _this_ repository) picks up the latest _tag_ (not master), so you'd get a really old codebase (17.05); does it work if you point it at the https://github.com/docker/engine repository? (that should get the latest 17.09 codebase)

Reply:
its become very weird and frustrating, and so i need to make a simple example, but the Sirupson one is when I _only_ import "github.com/docker/engine" and try to get go mod to use v18.09.1
somewhere in the deps I have it ends up adding "github.com/docker/docker" and then things go boom.